README.md

```diff
@@ -64,34 +84,51 @@ The artifacts can be obtained by:
     <type>war</type>
 </dependency>
+
 ~~~
+
 and Alfresco maven repository:
-~~~
+
+~~~xml
+
 <repository>
   <id>alfresco-maven-repo</id>
   <url>https://artifacts.alfresco.com/nexus/content/groups/public</url>
 </repository>
+
 ~~~
+
 The SNAPSHOT versions of the artifact are not published.
 
 ## Setting up and building your development environment
-See the [Development Tomcat Environment](https://github.com/Alfresco/acs-community-packaging/tree/master/dev/README.md)
-page which will show you how to try out your repository changes in a local tomcat instance.
-If you wish to use Docker images, take a look at the aliases ending in `D` and the docker-compose files in this
-project's test modules.    
+
+See the [**Development Tomcat Environment**](https://github.com/Alfresco/acs-community-packaging/tree/master/dev/README.md)
+page which will show you how to try out your repository changes in a local Tomcat instance or using Docker containers. 
+
+## Connection with Alfresco Content App
+After you have set up correctly your environment you may want to connect it with Alfresco Content Application.
+To do this - see this [**link**](https://github.com/Alfresco/alfresco-content-app).
 
 ## Branches
-This project has a branch for each ACS release. For example the code in ACS 6.2.1 is a
-branch called `releases/6.2.2`. In addition to the original 6.2.2 release it will also contain Hot Fixes
-added later. The latest unreleased code is on the `master` branch. There are also `.N` branches, such as 
-`releases/7.1.N` on which we gather unreleased fixes for future service pack releases. They do not indicate
+
+This project has a branch for each ACS release. For example the code in ACS 6.2.2 is a
+branch called **`release/6.2.2`**. In addition to the original 6.2.2 release it will also contain Hot Fixes
+added later. The latest unreleased code is on the **`master`** branch. There are also **`.N`** branches, such as 
+**`release/7.1.N`** on which we gather unreleased fixes for future service pack releases. They do not indicate
 that one is planned.
 
 For historic reasons the version of artifacts created on each branch do not match the ACS version.
-For example artifact in ACS 7.2.0 will be `14.<something>`.
+For example artifact in ACS 7.2.0 will be **`14.<something>`**.
 
-The enterprise projects which extend the `alfresco-community-repo` use the same branch names and leading
+The enterprise projects which extend the **`alfresco-community-repo`** use the same branch names and leading
 artifact version number.
 
-### Contributing guide
-Please use [this guide](CONTRIBUTING.md) to make a contribution to the project.
+## Contributing guide
+
+Please use [**this guide**](CONTRIBUTING.md) to make a contribution to the project.
+
+## Helpful links
+
+- [Alfresco Content Services Documentation ](https://docs.alfresco.com/content-services/latest/)
+- [Alfresco Content Application Repository](https://github.com/Alfresco/alfresco-content-app)
```

Comment:
```suggestion
```

There's many optional components that can be integrated with ACS. I wouldn't go out of our way to suggest on how to integrate with ACA here, so I'd remove this link + the related section.

Reply:
Hi, I understand that ACA is optional and not necessary for every use - I reedited what you've suggested and I think everything is OK now.